packages/game/src/rules/variants/sudoku.ts

```diff
@@ -71,8 +70,8 @@ export function sudokuWalkUpAll(
     if (allDiscardedSet.has(currentRank)) {
       // We hit a new dead rank.
       for (const writeRank of eRange(lastDead + 1, currentRank)) {
-        const maxScore = currentRank - writeRank;
-        maxScoresForEachStartingValueOfSuit[writeRank - 1] = maxScore;
+        maxScoresForEachStartingValueOfSuit[writeRank - 1] =
```

Comment:
why are you removing my SLR change here?

Reply:
Linting suggested it, but I don't have a real preference on it.

Reply:
what lint suggested it?

Reply:
Oh I guess it's not linting rules, it's my IDE itself, I'll revert it and re-configure.

Reply:
sounds like a terrible feature, its optimizing for the "smallest number of expressions", which is usually the exact opposite of "easiest code to read"